algorithms/Centrality/tg_category_centrality.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
suggest to use `yaml` as file extension and change pattern to use yaml extension.

Reply:
My understanding is that `.yml` is more widely recognized by third party editors (because of an antiquarian attachment to three letter extensions haha). I can certainly change it, but why do you prefer `.yaml`?

Reply:
you are right,  use yml and keep it consitent.